### PR TITLE
Modes update

### PR DESCRIPTION
**Pull request by BryonLewis**
- Updates Modes to be named "Analyst" and "Annotator"
- Uses local storage to store the DB Source Information, either RD-WATCH or Scoring to indicate what datasource is being used
- Creates new Component ModeSelector.Vue to be used to change modes
- The settings panel added the option to change the data source.
- Defaults to using the RD-WATCH datasource.

image

### CODE REVIEW

**Comment by BryonLewis** [comment]
TASK:
Disable + Add Proposal in Scoring Proposal view.

File: vue/src/components/LayerSelection.vue [hunk 1]
```diff
@@ -547,7 +547,7 @@ const addProposal = () => {
       :color="state.filters.addingSitePolygon ? 'primary' : ''"
       @click="addProposal()"
     >
-      <v-icon>mdi-plus</v-icon>Add Proposal
+      <v-icon>mdi-plus</v-icon>Add Proposal {{ scoringApp }}
```
Comment (mvandenburgh) [inline vue/src/components/LayerSelection.vue:550]: Debug leftover?
```suggestion
      <v-icon>mdi-plus</v-icon>Add Proposal
```

**Review by mvandenburgh — CHANGES_REQUESTED** [review]
LGTM just one removal I assume from debugging

File: vue/src/components/LayerSelection.vue (hunk 1 above)
Comment (BryonLewis) [reply vue/src/components/LayerSelection.vue]: Exactly, it was something from yesterday before the demo to remove the Add Proposal from the scoring view.  I've updated it.

**Review by mvandenburgh — APPROVED** [review]

**Pull request merged**